docs/Change_Log.md

```diff
@@ -3,6 +3,12 @@
 >danger
 >API and Gateway versions below v6 will be discontinued on October 16, 2017, after which they will be non-functioning.
 
+## July 31, 2017 - Breaking Change
+
+After today, we are removing the existing concept of a default channel from Discord. We saw a use case in many servers where the previously-default #general channel was being repurposed as an announcement-only, non-writable channel for new members. In practice, server owners were using bots to clear the entire message history of the default channel in order to change its purpose. Now that channel can simply be deleted and re-created with the desired permissions.
```

Comment:
this first sentence is a bit misleading, perhaps we can clarify. we are not removing the concept of a default channel, just changing how "default" works.. so perhaps "changing how default channels function" might be better phrasing.

I also think we should drop the line about people using bots to clear the history, since the line before and after it offers enough context